e2e/pages/manage/recordArrivalPage.ts

```diff
@@ -12,13 +12,13 @@ export class RecordArrivalPage extends BasePage {
   }
 
   async recordArrival() {
-    const arrivalDateTime = addDays(new Date(), -1) // Yesterday same time
+    const arrivalDateTime = addDays(new Date('2025-06-04T00:00'), -1) // Yesterday same time
```

Comment:
I don't think this should be a literal. With the new validation it would fail after a week!

Reply:
Absolute facepalm, this was for testing 🤦